test/unit/uk/gov/hmrc/tariffclassificationfrontend/controllers/HelloWorldControllerSpec.scala

```diff
@@ -27,8 +28,8 @@ import uk.gov.hmrc.tariffclassificationfrontend.config.AppConfig
 import uk.gov.hmrc.tariffclassificationfrontend.controllers.HelloWorld
 
 
-class HelloWorldControllerSpec extends WordSpec with Matchers with GuiceOneAppPerSuite {
-  val fakeRequest = FakeRequest("GET", "/")
+class HelloWorldControllerSpec extends WordSpec with Matchers with GuiceOneAppPerSuite with MockitoSugar {
```

Comment:
you are not using `Mockito` here, so I guess you can remove `MockitoSugar `

Reply:
Done